bedrock/events/tests/test_data/reps.ical

```diff
@@ -10,7 +10,7 @@ DTSTART;TZID=Europe/Skopje:20141007T180000
 DTEND;TZID=Europe/Skopje:20141007T190000
 LOCATION:KIKA\, Skopje\, Macedonia\, F.Y.R. of
 SUMMARY:Community meeting in Skopje
-DESCRIPTION:It&#39;s been a long summer and finally we have enough people back
+DESCRIPTION:It&#39;s been a long summer and finally we have enough people back
```

Comment:
This is just test data. It's what I got from reps.mozilla.org when I originally wrote the tests. I think it's a good test if it's still representative of what reps.m.o is sending us, but I've not checked in a while.